app/src/main/java/org/mifos/mobile/ui/about/AboutUsScreen.kt

```diff
@@ -0,0 +1,125 @@
+import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
+import org.mifos.mobile.R
+import org.mifos.mobile.core.ui.component.CopyrightItemContent
+import org.mifos.mobile.core.ui.component.MifosItemCard
+import org.mifos.mobile.core.ui.component.MifosItemContent
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+import org.mifos.mobile.ui.activities.PrivacyPolicyActivity
+import org.mifos.mobile.utils.Constants.LICENSE_LINK
+import org.mifos.mobile.utils.Constants.SOURCE_CODE_LINK
+import org.mifos.mobile.utils.Constants.WEBSITE_LINK
+import org.mifos.mobile.utils.StartActivity
+import java.util.*
+
+@SuppressWarnings("UnusedMaterial3ScaffoldPaddingParameter")
+@Composable
+fun AboutUsScreen(viewModel: AboutUsViewModel = viewModel()) {
+    val context = StartActivity(LocalContext.current)
+
+    MifosMobileTheme {
+        Scaffold {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(16.dp, 56.dp, 16.dp, 16.dp)
+            ) {
+                Image(
+                    painter = painterResource(id = R.mipmap.mifos_icon),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .size(100.dp)
+                        .fillMaxWidth()
+                        .align(Alignment.CenterHorizontally)
+                )
+
+                Text(
+                    text = stringResource(id = R.string.app_name),
+                    style = MaterialTheme.typography.headlineMedium,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(bottom = 8.dp)
+                )
+
+                Text(
+                    text = stringResource(id = R.string.about_app_description),
+                    style = MaterialTheme.typography.titleSmall.copy(),
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(bottom = 16.dp)
+                )
+
+                LazyColumn {
+                    items(viewModel.aboutUsList) { item ->
+                        MifosItemCard(onClick = {
+                            when (item.name) {
+                                R.string.official_website -> {
+                                    context.startActivity(WEBSITE_LINK)
+                                }
+                                R.string.licenses -> {
+                                    context.startActivity(LICENSE_LINK)
+                                }
+                                R.string.privacy_policy -> {
+                                    context.startActivity(PrivacyPolicyActivity::class.java)
+                                }
+                                R.string.sources -> {
+                                    context.startActivity(SOURCE_CODE_LINK)
```

Comment:
Create another variable inside the ViewModel named `AboutUsItemEvent`, you can use a liveData and StateFlow and then observe it inside the fragment. Based on the itemId navigate to the proper screens. I mean the navigation should be done inside the fragment not this composable.